src/transformers/onnx/features.py

```diff
@@ -102,6 +104,7 @@ class FeaturesManager:
             "multiple-choice": AutoModelForMultipleChoice,
             "question-answering": AutoModelForQuestionAnswering,
             "image-classification": AutoModelForImageClassification,
+            "masked-im": AutoModelForMaskedImageModeling,
```

Comment:
For reviewers: we picked `masked-im` as the name to match the pre-existing `masked-lm` used for language modeling